guides/hosting/performance/performance-tweaks.md

```diff
@@ -166,6 +166,8 @@ zend.detect_unicode=0
 realpath_cache_ttl=3600
 ```
 
+Also PHP PCRE Jit Target should be enabled, this can checked using `php -i | grep 'PCRE JIT Target'` or looking into phpinfo page.
```

Comment:
```suggestion
Also PHP PCRE Jit Target should be enabled, this can checked using `php -i | grep 'PCRE JIT Target'` or looking into the phpinfo page.
```